libraries/AP_UAVCAN/AP_UAVCAN_DNA_Server.cpp

```diff
@@ -580,6 +586,31 @@ void AP_UAVCAN_DNA_Server::handleNodeInfo(uint8_t node_id, uint8_t unique_id[],
     }
 }
 
+void AP_UAVCAN_DNA_Server::log_NodeStatus(uint8_t node_id, uint32_t uptime_sec, uint8_t healthy, uint8_t mode)
+{
+    if (node_id > MAX_NODE_ID) {
+        return;
+    }
+
+    /*
+      if we haven't logged this node then log it now
+     */
+    if (!logged_CANH.get(node_id) && AP::logger().logging_started()) {
+        logged_CANH.set(node_id);
```

Comment:
simplest way to make it handle a single dropped pkt without logging an error is to only log the error if the bit is already set in logged_CANH when the failure happens.
also, logged_CANH needs to be unset when we do get a good reply
that way we'll log unhealthy when 2 pkts are no replied to in a row, but mark healthy when one reply is received